aws-proofs/README.md

```diff
@@ -32,6 +33,8 @@ automatically spins up and terminates the corresponding AWS instance.
 - `skip_dups`:   skip duplicated proofs (default true).
                  Set to empty string to also run proofs that are already checked
                  in other proof sessions.
+- `simpl_export`: Whether to save C graph-lang for binary verification (optional).
+- `ci_branch`:   ci-actions branch to use on AWS VM (optional).
```

Comment:
I initially didn't put `ci_branch` into the action inputs because I was too lazy to think through the security implications, but I think it is Ok: if you can push to a branch, you can already do all the other things that would be dangerous.